test/integrations/destinations/google_adwords_enhanced_conversions/dataDelivery/oauth.ts

```diff
@@ -255,11 +264,11 @@ export const v1oauthScenarios = [
           output: {
             authErrorCategory: 'AUTH_STATUS_INACTIVE',
             message:
-              '""The caller does not have permission" during Google_adwords_enhanced_conversions response transformation"',
+              '"[{"error":{"code":403,"message":"The caller does not have permission","errors":[{"message":"The caller does not have permission","domain":"global","reason":"forbidden"}],"status":"PERMISSION_DENIED"}}] during Google_adwords_enhanced_conversions response transformation"',
```

Comment:
`"${JSON.stringify([{ error: { code: 403, message: "The caller does not have permission", errors: [{ message: "The caller does not have permission", domain: "global", reason: "forbidden" }], status: "PERMISSION_DENIED" }}])} during Google_adwords_enhanced_conversions response transformation"`

is this better?

Reply:
Didn't get it. Are you asking to use `JSON.stringify` in oauth.ts file?

Reply:
Here